openpype/hosts/maya/plugins/publish/validate_review.py

```diff
@@ -0,0 +1,35 @@
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import (
+    ValidateContentsOrder, KnownPublishError
+)
+
+
+class ValidateReview(pyblish.api.InstancePlugin):
+    """Validate review."""
+
+    order = ValidateContentsOrder
+    label = "Validate Review"
+    families = ["review"]
+
+    def process(self, instance):
+        cameras = cmds.ls(
+            instance.data["setMembers"], long=True, dag=True, cameras=True
+        )
+
+        # validate required settings
+        if len(cameras) == 0:
+            raise KnownPublishError(
+                "No camera found in review instance: {}".format(instance)
+            )
+        elif len(cameras) > 2:
+            raise KnownPublishError(
+                "Only a single camera is allowed for a review instance but "
+                "more than one camera found in review instance: {}. "
+                "Cameras found: {}".format(instance, ", ".join(cameras))
+            )
+
+        camera = cameras[0]
```

Comment:
This should actually also be validating `instance.data["review_camera"]` right to ensure that whatever might change in the collector with regards to how a camera is retrieved that it still correctly retrieves it?

We're now duplicating the logic and magically rely on the logic to stay in sync between here and the collector?